MODULE.bazel.lock

```diff
@@ -20,8 +20,8 @@
     "https://bcr.bazel.build/modules/aspect_rules_js/2.0.1/MODULE.bazel": "a09d32aa997ad899fcd88dc6e629188f03351994f0698bc32a9979c5bcfd5c31",
     "https://bcr.bazel.build/modules/aspect_rules_js/2.0.1/source.json": "62bda0d34f3817af165d1903b890ecbf6e50d652492f8bb83682636e997b5619",
     "https://bcr.bazel.build/modules/aspect_rules_lint/0.12.0/MODULE.bazel": "e767c5dbfeb254ec03275a7701b5cfde2c4d2873676804bc7cb27ddff3728fed",
-    "https://bcr.bazel.build/modules/aspect_rules_lint/1.0.0-rc10/MODULE.bazel": "43df678221c15a8d44a72f6496842238bd9612c97d024c145ef97c776534af0b",
-    "https://bcr.bazel.build/modules/aspect_rules_lint/1.0.0-rc10/source.json": "5e55a8d8b68f2f3412d2bd06e53eebed70e811a7681819305275398e90542768",
+    "https://bcr.bazel.build/modules/aspect_rules_lint/1.0.0-rc8/MODULE.bazel": "7c119c96df01e97ed8501ad319a26f29c975e22121d06fae3e88be5d0a0e96ee",
+    "https://bcr.bazel.build/modules/aspect_rules_lint/1.0.0-rc8/source.json": "0c87d888e7b38d2cb2cf051003579fb75ab68889ef7d4f2ea22e800b262823c4",
```

Comment:
That looks like a downgrade?

Reply:
downgrade is intentional because Andrzej found a possible formater bug in rules_lint
https://github.com/aspect-build/rules_lint/issues/413

same for removed lines. trying to investigate the bug in rules_lint, because its not reproducible there, me or Andrzeh will get back in a few days to this PR once its more clear what's up with rules_lint behaviour